homeassistant/helpers/template.py

```diff
@@ -308,6 +308,8 @@ def __str__(self) -> str:
 class TupleWrapper(tuple, ResultWrapper):
     """Wrap a tuple."""
 
+    __slots__ = ()
```

Comment:
For the full potential, the `ResultWrapper` class should have a `__slots__` attribute as well.